rspec-sidekiq.gemspec

```diff
@@ -14,12 +14,14 @@ Gem::Specification.new do |s|
   s.add_dependency 'rspec-core', '~> 3.0', '>= 3.0.0'
   s.add_dependency 'sidekiq', '>= 2.4.0'
 
-  s.add_development_dependency 'rspec', '~> 3.0'
+  s.add_development_dependency 'rspec'
   s.add_development_dependency 'coveralls', '~> 0.8', '>= 0.8.0'
   s.add_development_dependency 'fuubar', '~> 2.0', '>= 2.0.0'
   s.add_development_dependency 'activejob', '~> 4.2', '>= 4.0.0'
   s.add_development_dependency 'actionmailer', '~> 4.2', '>= 4.0.0'
   s.add_development_dependency 'activerecord', '~> 4.2', '>= 4.0.0'
+  s.add_development_dependency 'activemodel', '~> 4.2', '>= 4.0.0'
+  s.add_development_dependency 'activesupport', '~> 4.2', '>= 4.0.0'
```

Comment:
On Travis it can't download rails 5 because of the version lock here. Maybe just remove it

Shall we add a gentile for rails 4.2 ? No reason to deprecate right now...

Then you might encounter a minimum Ruby version error with rails 5.2 and Ruby 2.2

Reply:
Removed the restrictions and added a Rails 4 Gemfile for Travis